.gitignore

```diff
@@ -70,3 +70,4 @@ ecr.repo
 #Helm
 **/Chart.lock
 /send-visit-create-message-sns.bash
+/send-visit-cancel-message-sns.bash
```

Comment:
wondering if these scripts could be checked in instead?

Reply:
yes, i think we need a better way, will look at checking in a template without tokens etc